File: code/packages/java/starlark-lexer/src/test/java/com/codingadventures/starlarklexer/StarlarkLexerTest.java
package com.codingadventures.starlarklexer;

import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertEquals;

class StarlarkLexerTest {
    @Test
    void pingReturnsPackageName() {
        assertEquals("starlark-lexer", new StarlarkLexer().ping());
    }
}

File: code/packages/kotlin/starlark-lexer/src/test/kotlin/com/codingadventures/starlarklexer/StarlarkLexerTest.kt
package com.codingadventures.starlarklexer

import kotlin.test.Test
import kotlin.test.assertEquals

class StarlarkLexerTest {
    @Test
    fun pingReturnsPackageName() {
        assertEquals("starlark-lexer", StarlarkLexer().ping())
    }
}
